hera_mc/cm_hookup.py

```diff
@@ -231,9 +204,13 @@ def __get_column_header(self, hup0):
                                                        exact_match=True)
         pr_key = cm_utils._make_part_key(hu.downstream_part, hu.down_part_rev)
         parts_col.append(get_part_type[pr_key]['part'].hptype)
+        parts_col.append('Start')
+        parts_col.append('Stop')
         return parts_col
 
     def __hookup_add_correlator_levels(self, hookup_dict, testing):
+        print("CM_HOOKUP[208]:  Adding correlator levels doesn't work with new pol hookup scheme yet.")
```

Comment:
I think this kind of thing should be a warning. In general, I'd like to avoid print statements that are warning about unexpected or adverse behavior -- they should really be warnings.